section1/set8/nestedLoops/main.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import "fmt"
+
+func main() {
+	const max = 5
+
+	fmt.Printf("%5s", "X")
+	for i := 0; i <= max; i++ {
+		fmt.Printf("%5d", i)
+	}
+	fmt.Println()
+
+	for i := 0; i <= max; i++ {
```

Comment:
please tell what is being achieved in this loop like printing the numbers to practice nested for loop

Reply:
Fixed and added for other for loops